crates/test-utils/src/deploy_helpers.rs

```diff
@@ -843,8 +844,12 @@ impl KakarotTestEnvironmentContext {
         self
     }
 
-    pub fn sequencer(&self) -> &TestSequencer {
-        &self.sequencer
+    pub fn sequencer(&self) -> Arc<KatanaSequencer> {
+        self.sequencer.sequencer.clone()
```

Comment:
We have this [pr](https://github.com/kkrt-labs/kakarot-rpc/pull/552) to remove blocking, not sure if that might affect this